db/migrate/20190912121620_create_conversations.rb

```diff
@@ -2,7 +2,7 @@ class CreateConversations < ActiveRecord::Migration[5.2]
   def change
     create_table(:conversations, &:timestamps)
 
-    create_table :conversation_users, bulk: true, id: false do |t|
+    create_table :conversation_users, bulk: true do |t|
```

Comment:
You can add a migration to remove the table and create it again if necessary. Instead of modifying the existing migration.

Reply:
Cool, for next time 😜